ckanext/odm_nav/lib/odm_nav_helper.py

```diff
@@ -171,6 +178,11 @@ def tag_for_topic(topic):
   tag_name = ''.join(ch for ch in topic if (ch.isalnum() or ch == '_' or ch == '-' or ch == ' ' ))
   return tag_name if len(tag_name)<=100 else tag_name[0:99]
 
+def check_top_topics_disabled(country_code):
+  for item in disabled_top_topic_links:
+    if item == country_code:
+      return True
```

Comment:
What I understand here is that you want to be able to specify certain country codes (i.e kh, vi,...) in a config variable named **ckan.odm_nav_concept.disable_top_topic_links_for_countries** in order to not show top topics menu in certain sites? If so, yes, looks consistant,

:+1: